server/README.md

```diff
@@ -74,7 +75,8 @@ with the Server Wrapper library.
 Triton Server C-API Wrapper is encapsulated in a shared library which is built
 from source contained in this repository. You can include the full
 capabilities by linking the shared library into your application and by using
-the C++ API defined in [server_wrapper.h](include/triton/developer_tools/server_wrapper.h).
+the C++ API defined in [generic_server_wrapper.h](include/triton/developer_tools/generic_server_wrapper.h)
```

Comment:
Why is it called "generic"? I guess at the end there will be another "server_wrapper" that follows JAVACPP restrictions? So the user will need to pick one of the `server_wrapper` because this `generic_server_wrapper` is not the complete class for use (not even a complete interface because it is missing some APIs, like infer API). I would probably not mention `generic_server_wrapper` and treat it to be utils (common parts) that internal to the different `server_wrapper` and the wrapper will extend it to provide the complete functionalities. So the user doesn't need to understand the relationship between "generic_wrapper" and the other, or even have the impression that they may implement something out of "generic_wrapper" by themselves.

Reply:
Yeah I think that makes more sense. Will just change docs to server_wrapper

Reply:
I think what Guan mentioned makes sense to me. Given that we're removing `generic_server_wrapper` from the docs to avoid confusion, I was wondering if it's possible to move some of the structure, `LoggingOptions` and `HostPolicy` for example, to maybe `common.h` file so that users will be able to use those classes without knowing anything related to `generic_server_wrapper`. Does it make sense?